config/sync/migrate_plus.migration.group_node_easychart.yml

```diff
@@ -56,11 +40,14 @@ process:
       plugin: migration_lookup
       migration: node_easychart
       source: uuid
-  uid: uid
+  uid:
+    -
+      plugin: d7_user_lookup
+      source: node_uid
```

Comment:
Easy to miss this - need to check what the field ids are in the source plugin when this happens. I kept getting NULL for uid and was very confused for a while.